src/main/flight/imu.c

```diff
@@ -576,7 +564,7 @@ static int calculateThrottleAngleCorrection(void)
 
 void imuUpdateAttitude(timeUs_t currentTimeUs)
 {
-    if (sensors(SENSOR_ACC) && acc.isAccelUpdatedAtLeastOnce) {
+    if (sensors(SENSOR_ACC) && acc.isAccelUpdatedAtLeastOnce && gyroIsCalibrationComplete()) {
```

Comment:
I thought so too. However I had problems where the estimated attitude would end up way off if the quad was moved when connecting the power source and for perhaps 5 seconds after. It's almost as if there are some huge jump in the gyro data during or after calibration.
Adding this check removed the issue.
Might be worth looking into further, but currently this check is needed.

Reply:
The gyroADC and gyroADCf values won't be populated unless `gyroCalibrationCompleted` is True, see https://github.com/betaflight/betaflight/blob/d5d3ee88beb63ded0f9c632c259cc31b0e9ad58e/src/main/sensors/gyro.c#L417
The estimate covariance is already initiated to the maximum value at start, and converges quickly once it's getting data. However, the covariance should probably reset each time `gyroCalibrationCompleted` is false, to avoid weirdness if another calibration is initiated somehow.

Reply:
(Or simply use gyro data during calibration ... )

Reply:
As I wrote before:
> The gyroADC and gyroADCf values won't be populated unless gyroCalibrationCompleted is True

Using uncalibrated data would be fine, using zeroed data will cause issues. Once the calibration is completed the filter converges so fast that it doesn't really matter anyways.